staff.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Staff
+---
+
+## Identifying Staff
+
+The _staff_ are the volunteers who act as network and system administrators to
```

Comment:
I think /about is meant to explain this. I'd expand the section at the bottom of /about with `/who libera/staff/*` and remove this entire h2 section

Reply:
Agreed; I'll remove the staff list.

Reply:
This all started as trying to resolve the ambiguity of an instruction to "contact staff", without saying either how to find them or how to contact them.

Perhaps this doesn't need to be its own whole page, but I feel a summary of the steps involved should go somewhere so that it can be referred to.